File: AarDemo/library-core/src/main/java/org/techbooster/sample/library_core/LibraryCoreActivity.java
package org.techbooster.sample.library_core;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

public class LibraryCoreActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_librarycore);
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
    }

    public static Intent createIntent(Context context) {
        return new Intent(context, LibraryCoreActivity.class);
    }
}

File: AarDemo/library-core/src/main/java/org/techbooster/sample/library_core/LibraryCoreActivity.kt
package org.techbooster.sample.library_core

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.Toolbar
import android.view.View

class LibraryCoreActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_librarycore)
        val toolbar = findViewById<View>(R.id.toolbar) as Toolbar
        setSupportActionBar(toolbar)
    }

    companion object {

        fun createIntent(context: Context): Intent {
            return Intent(context, LibraryCoreActivity::class.java)
        }
    }
}
